var/spack/repos/builtin/packages/mrcpp/package.py

```diff
@@ -13,15 +13,27 @@ class Mrcpp(CMakePackage):
     error control in numerical computations."""
 
     homepage = "https://mrcpp.readthedocs.io/en/latest/"
-    url = "https://github.com/MRChemSoft/mrcpp/archive/v1.1.0.tar.gz"
+    url = "https://github.com/MRChemSoft/mrcpp/archive/v1.3.5.tar.gz"
 
     maintainers = ["robertodr", "stigrj", "ilfreddy"]
 
-    version('1.2.0-alpha2',
-            sha256='8f4df594751a5b7e76b09a62450c6c4956b1974876afa143cc9b5703156ccd40')
-    version('1.1.0',
-            sha256='e9ffb87eccbd45305f822a0b46b875788b70386b3c1d38add6540dc4e0327ab2',
+    version('1.3.5',
+            sha256='3072cf60db6fa1e621bc6e6dfb6d35f9367a44d9d312a4b8c455894769140aed',
             preferred=True)
```

Comment:
`preferred=True` is no longer needed, you can remove that. Also, if you want to shorten the package, you can keep all `version()` directives on a single line. They are excluded from flake8 line-length checks.